tests/4.5/target_enter_exit_data/test_target_enter_exit_data_if.c

```diff
@@ -37,10 +38,13 @@ int main() {
   }
 
   // We test for offloading
-#pragma omp target map(from: isOffloading)
-  {
-    isOffloading = !omp_is_initial_device();
-  }
+  int isOffloading; 
+  OMPVV_TEST_AND_SET_OFFLOADING(isOffloading);
+ 
+  if (!isOffloading)
+  OMPVV_WARNING("It is not possible to test conditional data transfers "
+                 "if the environment is shared or offloading is off. Not testing "
+                 "anything"); 
 
   // check multiple sizes. 
   for (size = 256; size <= 1024; size += 256) {
```

Comment:
I'm not so sure about this test now. What we want to check if tarrget enter exit data if, not target if. But I can't think of a way that we could do this without getting into shared data environment discussions.

The test should be checking if the if clause in the enter and exit data directives have any effect in the semantics.